### ISSUE TITLE
Can a CityJSONFeature be of a 2nd-level type?

### ISSUE THREAD

**Issue by hugoledoux** [cityjson/specs#169]
I realised this is not 100% clear in the specs, should this below be allowed? 

Notice the `BuildingPart` type. This would imply that somewhere in the stream (before or after I guess) there is another `CityJSONFeature` with `"id"=id-99` and that the client must handle this case. This makes the life of the creator of the stream easier, but complicates that of the receiver (the features are not independent anymore).

I am ambivalent about this, curious about others opinions here! 

```json
{
  "type": "CityJSONFeature",
  "id": "id-1", 
  "CityObjects": {
    "id-1": {
      "type": "BuildingPart", 
      "attributes": { 
        "roofType": "gabled roof"
      },
      "parents": ["id-99"],
      "geometry": [...]
    }
  },
  "vertices": [...]
}
```

**Comment by balazsdukai** [comment]
It says in the specs that _"all the children of the "CityJSONFeature" must be included (and the children of the children (recursively), if there are any)."_
To this means that all related 2nd level objects must be included with the parent in the same feature.

**Comment by clausnagel** [comment]
Good catch, @balazsdukai. Having the building part twice in the stream as in your example would be redundant and quite confusing.

I'd also say that a CityJSONFeature should be a 1st-level object with all its children included, but not a 2nd-level object. Makes consuming the stream easier and I don't really see an issue on the producer side.

**Comment by cportele** [comment]
My preference would also also be to include children always together with the 1st-level object.

**Comment by hugoledoux** [comment]
OK, thanks for the prompt replies everyone!

We stick to what was there.
